processing/src/test/java/org/apache/druid/java/util/common/DateTimesTest.java

```diff
@@ -123,4 +126,28 @@ public void testCanCompareAsString()
         DateTimes.of("2000").withZone(DateTimes.inferTzFromString("America/Los_Angeles")))
     );
   }
+
+  @Test
+  public void testMillisElapsed() throws Exception
+  {
+    Stopwatch stopwatch = Stopwatch.createStarted();
+    Thread.sleep(100);
```

Comment:
Instead of sleep, we can use a custom ticker to advance time artificially. 
`FakeTicker` is a subclass which can be used to 

```
FakeTicker fakeTicker = new FakeTicker();
Stopwatch stopwatch = Stopwatch.createStarted(fakeTicker);
fakeTicker.advance(100, TimeUnit.MILLIS);
```

Reply:
Yeah, good idea.